SolizWilson/colecciones/src/main.java

```diff
@@ -0,0 +1,44 @@
+import java.util.Scanner;
+
+public class main {
+
+    private static int i;
+
+    public static void main(String[] args) {
+
+        Scanner leer = new Scanner(System.in);
+
+
+        Cliente a[] = new Cliente[2];
```

Comment:
As a practice is ok to use basic Arrays but as you saw on the fifth lecture of Udacity there wrappers for this types of objects which are called collections please use those the next time